package.json

```diff
@@ -51,7 +51,7 @@
     "lint:changelog:rc": "auto-changelog validate --rc",
     "lint:eslint": "eslint . --ext js,ts,tsx,snap --cache",
     "lint:eslint:fix": "yarn lint:eslint --fix",
-    "lint:lockfile": "lockfile-lint --path yarn.lock --allowed-hosts npm yarn github.com codeload.github.com --empty-hostname false --allowed-schemes \"https:\" \"git+https:\"",
+    "lint:lockfile": "lockfile-lint --path yarn.lock --allowed-hosts npm yarn github.com codeload.github.com --empty-hostname true --allowed-schemes \"https:\" \"git+https:\" \"npm:\" \"patch:\" \"workspace:\"",
```

Comment:
empty hostname was required due to my changes, for one because the first entry in the lockfile is now a metadata key that does not contain a host, but possibly also due to the naive logic of splitting the host name from the resolution.